### ISSUE TITLE
[Question About Profiler] Is there a way to know which number of layer of neural network the tensor belongs to in the result of MXNet profiller? 

### ISSUE THREAD

**Issue by Ricahrd-Li** [apache/incubator-mxnet#17663]
## Description
(A clear and concise description of what the feature is.)
- If the proposal is about a new model, provide description of what the model is.
- If the proposal is about an API, provide mock examples if possible.

Hi there! 
Because of need in work, I am wondering is there a way to know **which number of layer of the neural network** the tensor belongs to in the result of MXNet profiller? 

For example,
In the result of profiler, I see result like this:
```
   {
        "name": "Convolution",
        "cat": "operator",
        "ph": "B",
        "ts": 1581521825791464,
        "pid": 16,
        "tid": 3803165864448931469
    }
,
    {
        "name": "Convolution",
        "cat": "operator",
        "ph": "E",
        "ts": 1581521825791916,
        "pid": 16,
        "tid": 3803165864448931469
    }
,

    {
        "name": "BatchNorm",
        "cat": "operator",
        "ph": "B",
        "ts": 1581521825791920,
        "pid": 16,
        "tid": 3803165864448931469
    }
```
I want to specify the number of layer in the profiling result, like:
```
   {
        "name": "Convolution_2",
        "cat": "operator",
        "ph": "B",
        "ts": 1581521825791464,
        "pid": 16,
        "tid": 3803165864448931469
    }
,
    {
        "name": "Convolution_2",
        "cat": "operator",
        "ph": "E",
        "ts": 1581521825791916,
        "pid": 16,
        "tid": 3803165864448931469
    }
,

    {
        "name": "BatchNorm_3",
        "cat": "operator",
        "ph": "B",
        "ts": 1581521825791920,
        "pid": 16,
        "tid": 3803165864448931469
    }
```
From the result above, I will know they are the second convolution layer and third batchnorm layer in my DNN (which may have 10 convolution layers and 5 batchnorm layers with different tensor shape and forward & backword computation time)

In the profiler of Horovod (called **Horovod timeline**), I am able to modify the source code and get "tensor_name" printed in the profiling result, which contains the number of layer. But in MXNet, I am still wondering how to achieve this. 

Thanks!

## References
- list reference and related literature
- list known implementations
